packages/utils/index.ts

```diff
@@ -36,10 +39,51 @@ export const bufferToJSON = (metadata: Uint8Array) => {
   return JSON.parse(new TextDecoder().decode(metadata));
 };
 
+export const decodeBuffer = (buffer: Uint8Array) => {
+  const textDecoder = new TextDecoder();
+  return textDecoder.decode(buffer);
+};
```

Comment:
Please use the `function` since all the rest functions delcared in this file use it. Also some comment describe what this function does would be helpful. `decodeBuffer` to me sounds like it should unpack a buffer and return a new buffer.